pkg/cmd/admin/registry/registry.go

```diff
@@ -334,3 +334,33 @@ func RunCmdRegistry(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg
 	fmt.Fprintf(out, "Docker registry %q service exists\n", name)
 	return nil
 }
+
+func generateProbeConfigForRegistry(ports []kapi.ContainerPort) *kapi.Probe {
+	healthzPort := defaultHealthzPort
+	if len(ports) > 0 {
+		healthzPort = ports[0].ContainerPort
```

Comment:
If we only ever bother passing in a list so that the first member of it can over-ride the default, why not just make the method signature
```
func generateProbeConfigForRegistry(port int) *kapi.Probe
```